books-i-read.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My Curriculum Vitae</title>
+    <meta charset="utf-8">
+	<meta name="Description" content="Curriculum Vitate">
+	<meta name="keywords" content="Name, Address, Education, Hobbies">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <article>
+	    	<h2>About The Books I Read.</h2>
+	    	<ul>

```

Comment:
you had options here. you could have done this without a list at all. There's nothing wrong with just a <section> and some <article>. But it's fine to use a <ul> in this case because you're telling browsers and search engines this is a list.